packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -305,19 +305,6 @@ def test_dcos_diagnostics_units_unit_nodes_node(dcos_api_session):
                 assert node_response['help'], 'help field cannot be empty'
 
 
-@pytest.mark.supportedwindows
```

Comment:
Why don't we need this test, anymore?

Reply:
We removed this endpoint since it was only checking for dcos-history status which we do not use anymore nor used before. We also improperly use dcos-history because instead of API we are messing with dcos-history files